examples/models/llama2/eval_llama_lib.py

```diff
@@ -38,7 +38,9 @@ def __init__(
         super().__init__()
         self._model = model
         self._tokenizer = tokenizer
-        self._device = torch.device("cpu")
+        self._device = (
+            torch.device("cuda") if torch.cuda.is_available() else torch.device("cpu"),
```

Comment:
nit: probably just need to do one check for device in the file and just use that

Reply:
I considered that, but that forces us to curry around a device param.

The device check in eval_llama actually gets refactored out (in a later PR) too

Reply:
I mean just a global variable, we can do it later